tools/doc/allhtml.mjs

```diff
@@ -31,31 +31,13 @@ for (const link of toc.match(/<a.*?>/g)) {
   // Split the doc.
   const match = /(<\/ul>\s*)?<\/\w+>\s*<\w+ id="apicontent">/.exec(data);
 
-  // Get module name
-  const moduleName = href.replace(/\.html$/, '');
-
   contents += data.slice(0, match.index)
-    .replace(/[\s\S]*?id="toc"[^>]*>\s*<\w+>.*?<\/\w+>\s*(<ul>\s*)?/, '')
-    // Prefix TOC links with current module name
-    .replace(/<a href="#(?!DEP[0-9]{4})([^"]+)"/g, (match, anchor) => {
-      return `<a href="#${moduleName}_${anchor}"`;
-    });
+    .replace(/[\s\S]*?id="toc"[^>]*>\s*<\w+>.*?<\/\w+>\s*(<ul>\s*)?/, '');
```

Comment:
The changes in this file look quite unrealted. Are these a side effect of a rebase?

Reply:
Yep, sorry. I'll fix that soon. 😅